docs/user/algo_bc.md

```diff
@@ -0,0 +1,16 @@
+# Behavioral Cloning
+
+[[Paper](https://arxiv.org/abs/1605.08478)] [[Implementation](https://github.com/rlworkgroup/garage/blob/master/src/garage/torch/algos/bc.py)]
+
+Garage's bahavioral cloning algorithm is based on model-free imitation learning with policy optimization. The algorithm uses a stochastic policy or deterministic policy and learns based on sample trajectories from the expert. It operates on SGD and is garanteed to converge to local minimum.
+
+Tuned examples: [bc_point](https://github.com/rlworkgroup/garage/blob/master/examples/torch/bc_point.py), [bc_point_deterministic_policy](https://github.com/rlworkgroup/garage/blob/master/examples/torch/bc_point_deterministic_policy.py)
+
+## Default Parameters
```

Comment:
It would be good for this document to call out the `batch_size` argument to the algorithm (as separate from the `batch_size` argument to the `LocalRunner`), and that each such batch will be split up into `minibatches_per_epoch` optimization steps.

Reply:
What will be the default value you' suggest for `batch_size ` in this case? I saw various values used in the examples i.e. 1000, 600, 400 with `policy_lr = 1e-2` etc

Reply:
Honestly I don't know, it depends on how complex the task is, which is why the algorithm doesn't provide a default. A few hundred is probably good for most tasks. If `batch_size / minibatches_per_epoch` is too small, then the neural network library overhead will make training too inefficient. If it's too big, then SGD will converge very slowly and the implicit regularization of SGD won't work as well.